mobile/src/full/java/org/openhab/habdroid/core/CloudMessagingHelper.kt

```diff
@@ -70,4 +72,12 @@ object CloudMessagingHelper {
         }
         else -> context.getString(R.string.info_openhab_gcm_connected)
     }
+
+    // Used to clear all notifications from statusbar
+    fun clearAllNotifications(context: Context?) {
+        val nm = context?.getSystemService(NOTIFICATION_SERVICE) as NotificationManager
+        nm.cancelAll()
```

Comment:
As mentioned before, this doesn't cut it. We have more notifications than just GCM ones, which must not be hidden by this.
Also need to update the foss variant of CloudMessagingHelper.

Reply:
fossStableDebug right ?

Reply:
Yeah, any foss build will work.

Reply:
I can do the foss part if you like.